integration_test/sojourn/test_helper.exs

```diff
@@ -1,4 +1,7 @@
-ExUnit.start([capture_log: :true, assert_receive_timeout: 500])
+ExUnit.start([capture_log: :true, assert_receive_timeout: 500,
+              exclude: [:queue_disconnected]])
+
+
```

Comment:
Thanks

On 27 April 2016 at 06:15, José Valim <notifications@github.com> wrote:

> In integration_test/sojourn/test_helper.exs
> <https://github.com/fishcakez/db_connection/pull/34#discussion_r61204832>:
>
> > @@ -1,4 +1,7 @@
> > -ExUnit.start([capture_log: :true, assert_receive_timeout: 500])
> > +ExUnit.start([capture_log: :true, assert_receive_timeout: 500,
> > +              exclude: [:queue_disconnected]])
> > +
> > +
>
> Extra new lines? :P
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/fishcakez/db_connection/pull/34/files/7a9fef74d3ea8983644a3c201777ce25d89b3b39#r61204832>
>